test/integration/setup.go

```diff
@@ -22,14 +22,10 @@ import (
 	"github.com/minishift/minishift/pkg/minikube/constants"
 	"io/ioutil"
 	"os"
-	"testing"
 )
 
-func setUp(t *testing.T) string {
-	testDir, err := ioutil.TempDir("", "minishift-integration-test-")
-	if err != nil {
-		t.Fatalf("Failed to create temp directory '%s'", testDir)
-	}
+func setUp() string {
+	testDir, _ := ioutil.TempDir("", "minishift-integration-test-")
```

Comment:
FYI, in the long run I want to get rid off ioutil.TempDir. Instead I want to create the test directories under the build folder in the source checkout. Using the tmp dir approach makes it really hard to find test files when one needs to debug something. Also it is sometimes better to keep the testdir until an explicit clean, instead of getting it deleted after the test run. I guess we can address this in another issue though.